### PR TITLE
[12.0] [MIG] purchase_request_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration to v12 of purchase_request_operating_unit merged with purchase_request_to_rfq_operating_unit

- [x] purchase_operating_unit https://github.com/OCA/operating-unit/pull/152

### CODE REVIEW

File: purchase_request_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Operating Unit in Purchase Requests",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "SerpentCS, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
```
Comment (nikul-serpentcs) [inline purchase_request_operating_unit/__manifest__.py]: https://github.com/OCA/operating-unit

File: purchase_request_operating_unit/model/purchase_request.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                                        'Operating Unit'))
+
+
+class PurchaseRequestLine(models.Model):
+    _inherit = 'purchase.request.line'
+
+    operating_unit_id = fields.Many2one(
+        'operating.unit',
+        related='request_id.operating_unit_id',
+        string='Operating Unit', readonly=True,
```
Comment (nikul-serpentcs) [inline purchase_request_operating_unit/model/purchase_request.py]: no need to ` readonly=True`

File: purchase_request_operating_unit/security/purchase_security.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2016-19 Eficent Business and IT Consulting Services S.L.-->
+<!--(http://www.eficent.com)-->
+<!--Copyright 2016-19 Serpent Consulting Services Pvt. Ltd.-->
+<!--(<http://www.serpentcs.com>)-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo noupdate="0">
```
Comment (nikul-serpentcs) [inline purchase_request_operating_unit/security/purchase_security.xml]: no need `noupdate="0"`

File: purchase_request_operating_unit/view/purchase_request_view.xml [hunk 4]
```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2016-19 Eficent Business and IT Consulting Services S.L.-->
+<!--(http://www.eficent.com)-->
+<!--Copyright 2016-19 Serpent Consulting Services Pvt. Ltd.-->
+<!--(<http://www.serpentcs.com>)-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo noupdate="0">
```
Comment (nikul-serpentcs) [inline purchase_request_operating_unit/view/purchase_request_view.xml]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improved COde

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing

File: purchase_request_operating_unit/__init__.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
```
Comment (nikul-serpentcs) [inline purchase_request_operating_unit/__init__.py]: Copyright 2016-17 ->  Copyright 2016-19

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Copyright in all file

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**